requirements.txt

```diff
@@ -1,6 +1,7 @@
 asdf>=2.3.0
 astropy>=3.0
 astroquery==0.3.9
+authlib==0.10
```

Comment:
Does this new dependency also need to be added to `setup.py`?

Reply:
@laurenmarietta It probably does, but there are probably a bunch of dependencies that we are missing in `setup.py` since we have been neglecting it, so perhaps updating `setup.py` deserves its own issue/PR (along the lines of the discussion in #261)